src/compo/viirs_l1bnc2ioda.py

```diff
@@ -273,13 +296,17 @@ def main():
         help="percentage of random thinning fro 0.0 to 1.0. Zero indicates"
         " no thinning is performed. (default: %(default)s)",
         type=float, default=0.0)
+    parser.add_argument(
+        '--secterm',
+        help="multiply sec of solar zenith angle to get true reflectance (Y/N)",
```

Comment:
and should change this comment
```suggestion
        help="presence of option will multiply secant of solar zenith angle to get true reflectance",
```